charts/neo4jv5-internal/values.yaml

```diff
@@ -49,7 +49,11 @@ neo4j:
       ]
 
   env:
-    NEO4JLABS_PLUGINS: '["apoc"]'
+    NEO4J_PLUGINS: '["apoc"]'
+
+  apoc_config:
+   apoc.trigger.enabled: "true"
+   apoc.import.file.enabled: "true"
```

Comment:
tested via preview env, we can correctly see apoc specific config in neo4j-preprod/neo4jv5-pr-108-apoc-config configmap

Data
====
apoc.conf:
----
apoc.import.file.enabled=true
apoc.trigger.enabled=true



do you want me to add this first in values.preprod.yaml ? before adding it to the common values ?